File: Praktikum_ALPRO/Pertemuan-3-Looping/Latihan 3/main.c
// Mencari nilai terkecil dari daftar angka input user

#include <stdio.h>
int angka_terkecil, angka;

int main(){
    printf("Masukan angka sebanyak 10 kali: ");
    scanf("%d", &angka_terkecil);

    for (int i=1; i<10; i++){
        printf("Masukan angka sebanyak 10 kali: ");
        scanf("%d", &angka);

        if (angka_terkecil>=angka){
            angka_terkecil = angka;
        }
    }
    printf("angka terkecil dari 10 bilangan: %d",angka_terkecil);
}

File: Praktikum_ALPRO/Pertemuan-3-Looping/Latihan 3/main.py
"""Mencari nilai terkecil dari daftar angka input user"""

angka_terkecil = int(input("Masukan angka sebanyak 10 kali: "))

for i in range (1, 10):
    angka = int(input("Masukan angka sebanyak 10 kali: "))
    if angka_terkecil>=angka:
        angka_terkecil=angka

print(f"angka terkecil dari 10 bilangan: {angka_terkecil}")
